### ISSUE TITLE
it is effectively not possible to use `APP PATHS` to register a short link to start code

### ISSUE THREAD

**Issue by jtnord** [microsoft/vscode#92156]
Note: #92080 was closed based on my "minimum reproduction case" rather than looking at the issue holistically so filing this hoilistic use case is broken bug.

- VSCode Version: 1.42.1
- OS Version: Windows 10 Pro

Steps to Reproduce:

1. register `code.exe` in the [`APP PATHS`](https://helgeklein.com/blog/2010/08/how-the-app-paths-registry-key-makes-windows-both-faster-and-safer/) registry key (set the location to be `c:\Program Files\Microsoft VS Code\Code.exe`
2. in a cmd.exe run `start code`
note how code dumps logging to the console, but is not blocking that console (see screenshot in #92080
3. close vs code
4. start code via `winkey+r` + `code`
note how code starts as expected.

5. close vs code

6. change the path to `code.exe` in `APP PATHS` to use `c:\Program Files\Microsoft VS Code\bin\Code.cmd`  (to not dump logs in cmd.exe)
7. in a cmd.exe run `start code`
note how no debug output appears in the console.
8. close vs code
9. start code via `winkey+r` + `code`
note how code starts but there is also a new command window started as well

Does this issue occur when all extensions are disabled?: Yes

TL;DR: as a user I expect to be able to launch code efficiently using standard OS mechanisms without being subject to extra console windows or debug messages polluting a console window.

**Comment by bpasero** [comment]
Sounds like https://github.com/microsoft/vscode/issues/68319 so I decided to mark it as duplicate. Please comment back if you disagree.
